supplementary_style_guide/main.adoc

```diff
@@ -2,6 +2,7 @@
 :doctype: book
 :toc: left
 :toclevels: 3
+:toc-title: Red Hat supplementary style guide
```

Comment:
Can you update this to the official title?

```suggestion
:toc-title: Red Hat supplementary style guide for product documentation
```

Reply:
I've adjusted the build to use the default value of {doctitle} for the navbar header. {toc-title} is used to give the TOC a custom title. PTAL at the updated build in https://file.emea.redhat.com/aireilly/ssg_updated_design/#

Reply:
Fixed the odd li>p indent issue and removed the :toc-title: attribute. The build uses {doctitle} for the navbar title now. Makes more sense. If we want to change the actual TOC title we can do that too. Up to you.

Reply:
See https://file.emea.redhat.com/aireilly/ssg_updated_design/#

Reply:
Thanks! Looks good now. I'm fine with the title being "Table of Contents" since that's how it shows in the current guide.

Reply:
It's already live on the mod docs site. Merge tomorrow during work day overlap?

Reply:
Just in case a quick fix is need on the GitHub actions side ...

Reply:
I chatted with Emily on this. I'll look and see if the rest of the style council is okay with moving forward with this, and then we can update as necessary if anything comes out of the accessibility testing.

Reply:
I have no idea about Mod Docs. I just noticed that it had been done.